src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -918,9 +919,9 @@ impl<'a, 'tcx> Visitor<'tcx> for Checker<'a, 'tcx> {
         debug!("visit_place: place={:?} context={:?} location={:?}", place, context, location);
         self.super_place(place, context, location);
         match *place {
-            Place::Base(PlaceBase::Local(_)) |
-            Place::Base(PlaceBase::Promoted(_)) => {}
+            Place::Base(PlaceBase::Local(_)) => {}
             Place::Base(PlaceBase::Static(ref global)) => {
+                assert!(global.promoted.is_none(), {});
```

Comment:
```suggestion
                assert!(global.promoted.is_none());
```

Reply:
The existing code has `Place::Base(PlaceBase::Promoted(_)) => {}`, so is removing the `{}` fine?

Reply:
yea. `{}` does nothing, it's just there to say "do nothing".

Reply:
But if I remove the `{}` and we get a `global.promoted.is_none() == False`, it would result in an ICE right? Are we okay with that?